docs/source/markdown/podman-artifact-push.1.md.in

```diff
@@ -0,0 +1,145 @@
+% podman-artifact-push 1
```

Comment:
you have to add the .md ones to `docs/source/markdown/.gitignore` so nobody commits them and the clean tree check does not fail in CI.